.github/workflows/generate_issues.yml

```diff
@@ -16,4 +16,4 @@ jobs:
       with:
         access-token: '${{ secrets.GITHUB_TOKEN }}'
         # This is to exclude workflows that will be searched in the nightly report.
-        exclude-workflow-files: 'performance-integration-tests.yml,mlmodeldownloader.yml'
+        exclude-workflow-files: 'performance-integration-tests.ymls'
```

Comment:
Nit: is the `s` at the end of `ymls` intentional?

Reply:
Nope. Good catch. Thanks!